test/gold/apps/chifra/chifra_run_abi.txt

```diff
@@ -2,59 +2,59 @@ chifra  abis 0xbb9bc244d798123fde783fcc1c72d3bb8c189413
+0x612e45a3	function	newProposal	newProposal(address,uint256,string,bytes,uint256,bool)	_recipient,_amount,_description,_transactionData,_debatingPeriod,_newCurator	_proposalID
+0x643f7cdd	function	DAOpaidOut	DAOpaidOut(address)	val_0	val_0
+0x674ed066	function	minQuorumDivisor	minQuorumDivisor()		val_0
+0x6837ff1e	function	newContract	newContract(address)	_newContract	
+0x70a08231	function	balanceOf	balanceOf(address)	_owner	balance
+0x73ad2a153c8b67991df9459024950b318a609782cee8c7eeda47b905f9baa91f	event	AllowedRecipientChanged	AllowedRecipientChanged(address,bool)	_recipient,_allowed	
+0x749f9889	function	changeAllowedRecipients	changeAllowedRecipients(address,bool)	_recipient,_allowed	_success
+0x78524b2e	function	halveMinQuorum	halveMinQuorum()		_success
+0x81f03fcb	function	paidOut	paidOut(address)	val_0	val_0
+0x82661dc4	function	splitDAO	splitDAO(uint256,address)	_proposalID,_newCurator	_success
+0x82bf6464	function	DAOrewardAccount	DAOrewardAccount()		val_0
+0x86abfce99b7dd908bec0169288797f85049ec73cbe046ed9de818fab3a497ae0	event	Voted	Voted(uint256,bool,address)	proposalID,position,voter	
+0x8b15a605	function	proposalDeposit	proposalDeposit()		val_0
+0x8c5be1e5ebec7d5bd14f71427d1e84f3dd0314c0f7b2291e5b200ac8c7c3b925	event	Approval	Approval(address,address,uint256)	_owner,_spender,_amount	
+0x8d7af473	function	numberOfProposals	numberOfProposals()		_numberOfProposals
+0x9046fefd66f538ab35263248a44217dcb70e2eb2cd136629e141b8b8f9f03b60	event	NewCurator	NewCurator(address)	_newCurator	
+0x96d7f3f5	function	lastTimeMinQuorumMet	lastTimeMinQuorumMet()		val_0
+0xa1da2fb9	function	retrieveDAOReward	retrieveDAOReward(bool)	_toMembers	_success
+0xa3912ec8	function	receiveEther	receiveEther()		val_0
+0xa9059cbb	function	transfer	transfer(address,uint256)	_to,_value	success
+0xb7bc2c84	function	isFueled	isFueled()		val_0
+0xbaac5300	function	createTokenProxy	createTokenProxy(address)	_tokenHolder	success
+0xbb28353e4598c3b9199101a66e0989549b659a59a54d2c27fbb183f1932c8e6d	event	Refund	Refund(address,uint256)	to,value	
+0xbe7c29c1	function	getNewDAOAddress	getNewDAOAddress(uint256)	_proposalID	_newDAO
+0xc9d27afe	function	vote	vote(uint256,bool)	_proposalID,_supportsProposal	_voteID
+0xcc9ae3f6	function	getMyReward	getMyReward()		_success
+0xcdef91d0	function	rewardToken	rewardToken(address)	val_0	val_0
+0xdbccb92686efceafb9bb7e0394df7f58f71b954061b81afb57109bf247d3d75a	event	CreatedToken	CreatedToken(address,uint256)	to,amount	
+0xdbde1988	function	transferFromWithoutReward	transferFromWithoutReward(address,address,uint256)	_from,_to,_value	success
+0xdd62ed3e	function	allowance	allowance(address,address)	_owner,_spender	remaining
+0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef	event	Transfer	Transfer(address,address,uint256)	_from,_to,_amount	
+0xdfc78bdca8e3e0b18c16c5c99323c6cb9eb5e00afde190b4e7273f5158702b07	event	ProposalTallied	ProposalTallied(uint256,bool,uint256)	proposalID,result,quorum	
```

Comment:
I wonder if brain-dead third-party code that parses this output will read the missing ending tab as an invalid line. The header suggests X fields in the file, this record (and others with no trailing tab) has X-1 fields.

Reply:
Hm, the header is on line 4 and has 6 items, all of the lines have 6 items, too. If the last is missing, it's empty, but the tab character is there. So I guess it should read it OK.